hack/run-origin-tests.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+. $(dirname "$0")/common.sh
+
+skopeo_workdir=$(mktemp -d)
+function finish_skopeo {
+  rm -rf ${skopeo_workdir}
+}
+trap finish_skopeo EXIT
+
+function get_ose_tests_binary {
+    # When running in a pod, we can't directly use the image.
+    # For this reason, we download the image and fetch the binary from the right layer
+    skopeo copy docker://"$ORIGIN_TESTS_IMAGE" oci:${skopeo_workdir}
+    echo "Fetching openshift-tests binary from $ORIGIN_TESTS_IMAGE"
+    for layer in ${skopeo_workdir}/blobs/sha256/*; do
+        echo "layer=${layer}"
+        set +e
+        testsbin=$(tar -t -f "$layer" | grep openshift-tests)
+        set -e
+        if [[ $testsbin ]]; then
+            echo "Found $testsbin on $layer"
+            tar xfv "$layer" "$testsbin"
+            mv "$testsbin" _cache/tools/openshift-tests
+            chmod +x _cache/tools/openshift-tests
+            rm -rf ${skopeo_workdir}
+            break
+        fi
+    done
+}
+
+CONTAINER_MGMT_CLI="${CONTAINER_MGMT_CLI:-docker}"
+TESTS_REPORTS_PATH="${TESTS_REPORTS_PATH:-/tmp/artifacts/}"
```

Comment:
this will not run over cnf-tests?

Reply:
no, the directory is created with '-p' flag and we just get one junit file in there. for example, 'junit_e2e_20210124-133651.xml'